handlers/dashboard.go

```diff
@@ -389,23 +389,27 @@ func getExecutionChartData(indices []uint64, currency string) ([]*types.ChartDat
 
 	var chartData = make([]*types.ChartDataPoint, len(blocks))
 	epochsPerDay := utils.EpochsPerDay()
+	lastFinalizedEpoch := services.LatestFinalizedEpoch()
 
 	for i := len(blocks) - 1; i >= 0; i-- {
-		consData := consMap[blocks[i].Number]
-		day := int64(consData.Epoch / epochsPerDay)
-		color := "#90ed7d"
-		totalReward, _ := utils.WeiToEther(utils.Eth1TotalReward(blocks[i])).Float64()
-		relayData, ok := relaysData[common.BytesToHash(blocks[i].Hash)]
-		if ok {
-			totalReward, _ = utils.WeiToEther(relayData.MevBribe.BigInt()).Float64()
-		}
+		blockEpoch := utils.TimeToEpoch(blocks[i].Time.AsTime())
+		if blockEpoch <= int64(lastFinalizedEpoch) {
```

Comment:
**Minor:** This would be a bit nicer to avoid nesting.
```suggestion
		if blockEpoch > int64(lastFinalizedEpoch) {
			continue
		}
```

Reply:
sure, refactored